examples/graphql-dataloader/graphql_dataloader.md

```diff
@@ -0,0 +1,22 @@
+# GraphQL service - Dataloader
```

Comment:
```suggestion
# GraphQL service - Dataloader
```
Check if `dataloader` is two words or one word.

Reply:
Should be `DataLoader`, as per the [original DataLoader documentation](https://github.com/graphql/dataloader).